templates/init/functions/typescript/_eslintrc

```diff
@@ -6,6 +6,7 @@ module.exports = {
   },
   extends: [
     "eslint:recommended",
+    "plugin:@typescript-eslint/recommended",
```

Comment:
Can you move this to after 'google'? This ruleset [turns off some common rules](https://github.com/typescript-eslint/typescript-eslint/blob/498f397ff3898dde631f37311615b555f38a414e/packages/eslint-plugin/src/configs/recommended.ts#L27) in favor of the plugin's version, but the 'google' ruleset is [turning those back on](https://github.com/google/eslint-config-google/blob/master/index.js#L167). As is, this ends throwing repetitive errors and warnings for the same issue in some cases.

Reply:
Done!